plugins/woocommerce-blocks/assets/js/blocks/product-filters/inner-blocks/removable-chips/frontend.ts

```diff
@@ -14,7 +14,11 @@ store( 'woocommerce/product-filter-removable-chips', {
 			const productFilterActiveStore = store< ProductFilterActiveStore >(
 				'woocommerce/product-filter-active'
 			);
-			return productFilterActiveStore.state.items;
+
+			return productFilterActiveStore.state.items.map( ( item ) => ( {
+				...item,
+				label: item.label.replace( /\s*\(\d+\)/, '' ),
```

Comment:
What do you think about adding `$` to the regexp to ensure we are matching it at the end of the string? It would make it a bit more specific:

```suggestion
				label: item.label.replace( /\s*\(\d+\)$/, '' ),
```

Reply:
Good suggestion!